config/routes.rb

```diff
@@ -12,4 +11,16 @@
   resources :books, only: :show do
     resources :reviews, only: :create
   end
+
+  resources :users, only: [] do
+    resources :orders, only: %i[index show] do
+      resources :order_items, only: :index
+    end
+  end
+
+  resource :order do
+    resources :order_items
+  end
+
+  resources :orders
```

Comment:
Why do you declare 3 times resourse/resources `order` and 2 times `order_items` in your routes?
Can we refactor this?